checklists/alz_checklist.en.json

```diff
@@ -1220,127 +1183,309 @@
+            "link": "https://learn.microsoft.com/azure/firewall/firewall-faq#why-does-azure-firewall-need-a--26-subnet-size"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Optimise Rule Configuration and processing",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "828cec2e-af6c-40c2-8fa2-1b681ee63eb7",
+            "id": "D07.12",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall-manager/rule-hierarchy"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use IP Groups or IP prefixes to reduce number of IP table rules",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "0da83bb1-2f39-49af-b5c9-835fc455e3d1",
+            "id": "D07.12",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/ip-groups"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Avoid wildcards as sources for DNATS",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "c44c6f0e-1642-4a61-a17b-0922f835c93a",
+            "id": "D07.13",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/tutorial-firewall-dnat"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Prevent SNAT Port exhaustion",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "7371dc21-251a-47a3-af14-6e01b9da4757",
+            "id": "D07.14",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/nat-gateway/tutorial-hub-spoke-nat-firewall"
+        },
+         {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Enable TLS Inspection",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "346840b8-1064-496e-8396-4b1340172d52",
+            "id": "D07.15",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#tls-inspection"
+        },
+        {
+            "category": "Network Topology and Connectivity",
```

Comment:
Can we make this just "Use web categories to allow or deny outbound access to specific topics."

The "consider" items are hard to execute as part of a review.